src/hooks/useEvidences.js

```diff
@@ -77,6 +78,7 @@ export default function useEvidences(rawEvidences) {
           })
         })
       )
+      setFetchingEvidences(false)
```

Comment:
I think we'll need to reset this to be `true` if it isn't already at the start of the `useEffect()`.

If a component using this hook re-renders with more evidence, and this was previously already set to false, it won't appear as syncing.